x/protocolpool/types/keys.go

```diff
@@ -6,6 +6,9 @@ const (
 	// ModuleName is the module name constant used in many places
 	ModuleName = "protocolpool"
 
+	// StreamModuleName is the name constant used for stream module account
+	StreamModuleName = "stream_module"
```

Comment:
```suggestion
	StreamAccount = "stream_acc"
```
since its not a module name, might be easier like this